src/commands/deploy/IDeployBaseContext.ts

```diff
@@ -14,6 +14,7 @@ export interface IDeployBaseContext extends ISubscriptionActionContext {
     targetContainer?: ContainerAppModel;
 
     imageSource?: ImageSourceValues;
+    buildType?: string;
```

Comment:
Also, we should try to document these context properties where necessary. How are these properties used and what do they control?